client/src/pages/Home.svelte

```diff
@@ -108,9 +119,31 @@
         column-gap: 0.75rem;
     }
 
+    footer {
+        position: absolute;
+        bottom: 0;
+        width: 100%;
+        text-align: center;
+        padding: var(--spacing-small);
+        color: var(--text-color);
+    }
+
     section {
         display: flex;
         justify-content: space-evenly;
         align-items: center;
     }
+
+    @media (max-height: 680px) {
+        article {
+            position: absolute;
+            top: 1rem;
+        }
+    }
+
+    @media (max-height: 620px) {
+        #techstack {
+            display: none;
+        }
+    }

```

Comment:
Is media queries the best way to do this? And if it is, we should just set it under one `max-height` value as i doubt `60px` would be much of a difference.

Reply:
You're right! There is a better solution. haha